src/App/App.test.tsx

```diff
@@ -152,6 +152,24 @@ describe('App', () => {
     expect(playlistToCheck).toBeInTheDocument();
   });
 
+  test('should be able to remove a saved a playlist while in the saved route and see it disappear', async () => {
+    (getGenres as jest.Mock).mockResolvedValue(mockGenres);
+    (getPlaylist as jest.Mock).mockResolvedValue(mockPlaylist1);
+    render(
+      <MemoryRouter>
+        <App />
+      </MemoryRouter>
+    );
+    const genreToClick: HTMLElement = await waitFor(() => screen.getByText('military western ska'))
+    userEvent.click(screen.getByText(genreToClick.innerHTML))
```

Comment:
Interesting to see that this statement is still selecting the genre correctly, even when the genres have been converted into buttons. Would it be better to target the role by button with a specific name or this way?